src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./PartnersAndSponsorsBlock.module.css";
+
+const LogoPropType = PropTypes.shape({
+  url: PropTypes.string.isRequired,
+  alt: PropTypes.string.isRequired,
+});
+
+const PartnersAndSponsorsBlock = ({ title, partnersAndSponsors }) => {
+  const GridAreaLeft = () => (
+    <div className={s.gridAreaLeft}>
+      <h1 className={s.title}>{title}</h1>
+    </div>
```

Comment:
This area appears simple enough to move into the return. Isn't that right @richardxia ?

Reply:
Although I would agree with that in isolation, because in this case `<GridAreaRight>` is more complex, I think it'd be better to keep this as a separate component for symmetry with `<GridAreaRight>`. I think that either we inline both of them or neither of them, and in this case, `<GridAreaRight>` looks complex enough to warrant being defined separately.